models/staging/jaffle_shop/stg_customers.sql

```diff
@@ -0,0 +1,8 @@
+with stg_customers as 
+(select

```

Comment:
in staging models, it's ideal to select * from your source in one CTE, then do your renaming in another CTE, so that you can separate logic modularly